app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -145,23 +221,59 @@ class SessionsViewPagerAdapter(
             is Tab.RoomTab -> {
                 RoomSessionsFragment.newInstance(tab.room)
             }
+            is Tab.TimeTab -> {
+                ScheduleSessionsFragment.newInstance(tab.schedule)
+            }
         }
     }
 
+    override fun getItemPosition(`object`: Any): Int {
+        return PagerAdapter.POSITION_NONE
```

Comment:
I have a question 👀 
Why is this method needed? 🙋

Reply:
This returned value will be used in Adapter class to determine whether or not it should re-create the ViewPager's current page when calling `notifyChanged`.
The default value of this method will keep the current page even if `notifyChanged` was called. But `PagerAdapter.POSITION_NONE` can re-create the current page.

<details>
This method need return either `PagerAdapter.UNCHANGED` or `PagerAdapter.POSITION_NONE`, and the default value is `PagerAdapter.UNCHANGED`.
`PagerAdapter.UNCHANGED` won't re-create the page, so it means a ViewPager *keep* the current page anyway regardless of its content was actually changed.
On the other hand, `PagerAdapter.POSITION_NONE` will re-create the current page. In this case,  `POSITION_NONE` is better. It's much more better to check the argument `object`, to be honest. 🙊 
</details>

Reply:
Thanks!
Can you add comment?
Example:
// For recreating Fragment

Reply:
sure thing 👍